spec/requests/users_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'rails_helper'
+
+RSpec.describe "Users", type: :request do
+
+  describe "User Registration" do
+    it "allows a user to sign up" do
+      post user_registration_path, params: {
+        user: {
+          user_name: "test",
+          phone: "1234567",
+          email: "testing@sample.com",
+          password: "password",
+          password_confirmation: "password"
+        }
+      }
+      expect(response).to redirect_to(root_path)
```

Comment:
When you're testing that a new user is being created, it's good to be a bit more thorough with what result you're testing for. Just testing that the response sends the user to the root_path doesn't quite ensure that the new record is being created when this action is finished, only that you were sent back to the root path after this request. I would suggest adding in an additional expect statement that shows that the user record was in fact created.